Changelog.md

```diff
@@ -19,6 +19,15 @@
 * Fix array: missing pointer touch could potentially cause use of freed memory.
 * Fix array: unnecessary additional allocation due to a bug
 
+### Notable Internal API Changes
+
+* `tapAsync` is moved from `Stream.Parallal` to `Stream.IsStream.Transform`. It
```

Comment:
typo `Parallal`. `tapAsync` was not moved, it was already present, and it already took a Fold. We can remove this bullet.